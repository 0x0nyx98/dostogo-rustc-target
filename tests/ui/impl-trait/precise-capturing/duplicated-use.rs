//@ revisions: real pre_expansion
//@[pre_expansion] check-pass

#![feature(precise_capturing)]
//~^ WARN the feature `precise_capturing` is incomplete

#[cfg(real)]
fn hello<'a>() -> impl Sized + use<'a> + use<'a> {}
//[real]~^ ERROR duplicate `use<...>` precise capturing syntax

fn main() {}
