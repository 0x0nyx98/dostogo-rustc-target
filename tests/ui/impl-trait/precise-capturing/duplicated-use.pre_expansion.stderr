warning: the feature `precise_capturing` is incomplete and may not be safe to use and/or cause compiler crashes
  --> $DIR/duplicated-use.rs:4:12
   |
LL | #![feature(precise_capturing)]
   |            ^^^^^^^^^^^^^^^^^
   |
   = note: see issue #123432 <https://github.com/rust-lang/rust/issues/123432> for more information
   = note: `#[warn(incomplete_features)]` on by default

warning: 1 warning emitted

